### DISCUSSION TITLE
I can't see aff log

### DISCUSSION THREAD

**Discussion by Bobshikalob** [Monika-After-Story/MonikaModDev#9098] [QA - tech support + general] [answered]
So I accidentally permanently deleted the files, then I got it back and everything was still the same, except for the log folder. It's not here anymore and I can't seem to find it anywhere. How do I get it back?

**Comment by Justformas** [comment]
How did you get everything back without the log folder? Or why can't you get the log folder back that way? Regardless, did you try making a new replacement folder titled "log" in your Doki Doki Literature Club MAS folder, and then running the game and doing something to affect your affection?

**Comment by Booplicate** [comment, marked as answer]
When you start the game the new logs will be created. The only way to get your old files is if you recover them.
